x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -1,11 +1,55 @@
 #!/bin/bash
 
+set -u
+
+fail() {
+  printf "%s\n" "$@" >&2
+  exit 1
+}
+
+# require bash
+if [ -z "${BASH_VERSION:-}" ]; then
+  fail "bash required"
+fi
+
+if [ $# -lt 5 ]; then
```

Comment:
If the "Auto download config" toggle is disabled, `autoDownloadConfig` is not present. Maybe we can sent `autoDownloadConfig=0` instead?

Reply:
good point, I was back and forth on this and making it required seems like the best option to me

Reply:
updated to not require for now so we don't get stuck on this.  If we want it later we can update the instructions in kibana at the same time